sale_commission/models/sale_commission.py

```diff
@@ -23,6 +23,12 @@
 from openerp import models, fields, api, exceptions, _
 
 
+PERIOD_MONTH = "monthly"
+PERIOD_QUARTER = "quaterly"
+PERIOD_SEMI = "semi"
+PERIOD_YEAR = "annual"
```

Comment:
Ok I see that you used it later, you could do another constant with the full list and keep those